services/QuillLMS/spec/models/concerns/teacher_spec.rb

```diff
@@ -551,6 +551,22 @@
     end
   end
 
+  context 'callbacks' do
+    describe '#ortto_newsletter_callback' do
+      let!(:teacher) { create(:teacher, send_newsletter: false) }
+
+      it 'should call NewsletterWorker when User.send_newsletter is changed' do
```

Comment:
I think a separate spec for testing changing of `send_newsletter` from true to false would be helpful.  Otherwise this feels like the worker is initiated more with a switch to true.